lib/theme_check/checks/img_lazy_loading.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module ThemeCheck
+  class ImgLazyLoading < HtmlCheck
+    severity :suggestion
+    categories :html, :performance
+    doc docs_url(__FILE__)
+
+    def on_img(node)
+      if node.attributes["loading"]&.value != "lazy"
```

Comment:
"eager" is also a valid value. Though we probably don't want to encourage this, we should permit it.